libkbfs/keybase_service_base.go

```diff
@@ -304,7 +310,14 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		return UserInfo{}, ConvertIdentifyError(assertion, err)
 	}
 
-	return k.processUserPlusKeys(res.Upk)
+	userInfo, err := k.processUserPlusKeys(res.Upk)
+	if err != nil {
+		return UserInfo{}, err
+	}
+
+	ei.userBreak(userInfo.Name, userInfo.UID, res.TrackBreaks)
```

Comment:
Maybe put a comment here that it's required to call this for every identify, even if `TrackBreaks` is `nil`.  That confused me at first.